blueprints/app/files/package.json

```diff
@@ -24,6 +24,8 @@
     "test:ember": "ember test"
   },
   "devDependencies": {
+    <% if (!typescript) { %>"@babel/eslint-parser": "^7.19.1",
```

Comment:
Approved, but it's failing because `eslint` was bumped in ember-cli's package file without updating the lock file.
I would omit that bump from this PR though.